docs/basics.md

```diff
@@ -759,12 +810,14 @@ Like in Mocha you can use `x` and `only` to skip tests or to run a single test.
 * `xScenario` - skips current test
 * `Scenario.only` - executes only the current test
 
-## Todo test
+## Todo
```

Comment:
Why `Todo test` was removed?

IMO, `Todo` looks like we forgot to add docs :)
How about `Not implemented tests (todo)`